net/FlatBuffers/ByteBuffer.cs

```diff
@@ -106,7 +116,9 @@ protected void WriteLittleEndian(int offset, int count, ulong data)
 
         protected ulong ReadLittleEndian(int offset, int count)
         {
+#if !BYTEBUFFER_NO_BOUNDS_CHECK
```

Comment:
Just ran the tests with just the method body optimized out; getting *roughly* the same timings as with the individual calls defined out (there's 0.02us or so in it). It's probably best to define out the body only for the sake of maintenance.